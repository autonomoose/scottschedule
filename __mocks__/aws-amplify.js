export const { Hub } = jest.requireActual('aws-amplify');

const mockDefaultCurrentSession = {
    idToken: { payload: {'custom:tenant': 'test'} },
    accessToken: { payload: {
            username: 'mockuname',
            sub: 'mocksub'
}}};



export const Auth = {
    currentSession: jest.fn(() => Promise.resolve(mockDefaultCurrentSession)),
    currentAuthenticatedUser: jest.fn(() => Promise.resolve({
        username: 'testuser1',
        deleteUser: jest.fn(() => Promise.resolve('deleted')),
    })),
    userAttributes: jest.fn(() => Promise.resolve([
        {Name: 'sub', Value: 'test'},
        {Name: 'email_verified', Value: 'true'},
        {Name: 'email', Value: 'testuser1@test.com'},
        {Name: 'custom:tenant', Value: 'test'},
        {Name: 'phone_number_verified', Value: 'false'},
        {Name: 'phone_number', Value: '111'},
    ])),
    updateUserAttributes: jest.fn(() => Promise.resolve()),
    verifyCurrentUserAttribute: jest.fn(() => Promise.resolve()),
    verifyCurrentUserAttributeSubmit: jest.fn(() => Promise.resolve()),
    signIn: jest.fn(() => Promise.resolve()),
    signOut: jest.fn(() => Promise.resolve()),
    changePassword: jest.fn((currUsr, oldpass, newpass) => Promise.resolve('changed')),
};

export const Storage = {
    configure: jest.fn(() => Promise.resolve('origMockfile')),
    get: jest.fn(() => Promise.resolve('origMockfile')),
    put: jest.fn(() => Promise.resolve('origMockfile')),
    remove: jest.fn(() => Promise.resolve('removed origMockfile')),
    list: jest.fn(() => Promise.resolve([
        {key: "public/", size: 0},
        {key: "public/testlabel1", size: 23470},
        {key: "public/testlabel2", size: 22270},
    ]))
}

// used by layout before every authenticated page
export const mockAPIgqlGetCurrentUser = {
    'data': {
        'getCurrentUser': {
            adisable: null, agroups: null,
            nname: "user3test - legacy primary",
            perm: "standard", userid: "mockgraphql"
}}};
export const API = {
    post: jest.fn(() => Promise.resolve({'message': 'passed'})),
    graphql: jest.fn(() => Promise.resolve(mockAPIgqlGetCurrentUser)),
};

