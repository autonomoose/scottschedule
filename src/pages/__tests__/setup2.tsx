import React from "react";
import { render, screen, waitFor } from "@testing-library/react";
import userEvent from '@testing-library/user-event'
import "@testing-library/jest-dom";

import { API } from 'aws-amplify';
import SetupStep2Page from "../setup2";

jest.mock('aws-amplify');

const mockEnqueue = jest.fn();
jest.mock('notistack', () => ({
  ...jest.requireActual('notistack'),
  useSnackbar: () => {
    return {
      enqueueSnackbar: mockEnqueue
    };
  }
}));

const mytest = <SetupStep2Page />;
describe("SetupStep2Page", () => {
  it("renders snapshot correctly", async () => {
    const {container} = render(mytest);
    await waitFor(() => {
        expect(screen.getByRole('button', {name: /sign out/i}))
        .toBeVisible();
    });

    expect(container.firstChild).toMatchSnapshot();
  });
  it("calls API correctly", async () => {
    const utils = render(mytest);
    await waitFor(() => {
        expect(utils.getByRole('button', {name: /sign out/i}))
        .toBeVisible();
    });

    const spy = jest.spyOn(API, 'post')
      .mockImplementation(() => Promise.resolve({'Response': 'completed'}));

    const acceptBox = utils.getByRole('checkbox', {name: /acceptbox/i});
    const submitButton = utils.getByRole('button', {name: /submit/i});
    expect(submitButton).toBeDisabled();

    userEvent.click(acceptBox);

    expect(submitButton).toBeEnabled();
    userEvent.click(submitButton);
    await waitFor(() => {
        expect(mockEnqueue).toHaveBeenCalledTimes(1);
    });
    // utils.debug(utils.getByRole('main'));

    spy.mockRestore();
  });

  it("calls API correctly w/ bad response", async () => {
    const utils = render(mytest);
    await waitFor(() => {
        expect(utils.getByRole('button', {name: /sign out/i}))
        .toBeVisible();
    });

    const spy = jest.spyOn(API, 'post')
      .mockImplementation(() => Promise.resolve({'Response': 'uncompleted'}));

    const acceptBox = utils.getByRole('checkbox', {name: /acceptbox/i});
    const submitButton = utils.getByRole('button', {name: /submit/i});
    expect(submitButton).toBeDisabled();

    userEvent.click(acceptBox);

    expect(submitButton).toBeEnabled();
    userEvent.click(submitButton);
    await waitFor(() => {
        expect(mockEnqueue).toHaveBeenCalledTimes(2);
    });
    // utils.debug(utils.getByRole('main'));

    spy.mockRestore();
  });
  it("calls API correctly but throws error", async () => {
    const utils = render(mytest);
    await waitFor(() => {
        expect(utils.getByRole('button', {name: /sign out/i}))
        .toBeVisible();
    });

    const spy = jest.spyOn(API, 'post')
      .mockImplementation((_apiname, _funname, _funparms) => Promise.reject({'Response': 'completed'}));

    const acceptBox = utils.getByRole('checkbox', {name: /acceptbox/i});
    const submitButton = utils.getByRole('button', {name: /submit/i});
    expect(submitButton).toBeDisabled();

    userEvent.click(acceptBox);

    expect(submitButton).toBeEnabled();
    userEvent.click(submitButton);
    await waitFor(() => {
        expect(mockEnqueue).toHaveBeenCalledTimes(3);
    });
    // utils.debug(utils.getByRole('main'));

    spy.mockRestore();
  });

  it("makes sure stupid box is checked", async () => {
    const utils = render(mytest);
    await waitFor(() => {
        expect(utils.getByRole('button', {name: /sign out/i}))
        .toBeVisible();
    });

    const spy = jest.spyOn(API, 'post')
      .mockImplementation(() => Promise.resolve({'Response': 'completed'}));

    const acceptBox = utils.getByRole('checkbox', {name: /acceptbox/i});
    const submitButton = utils.getByRole('button', {name: /submit/i});
    expect(submitButton).toBeDisabled();

    userEvent.click(acceptBox);

    expect(submitButton).toBeEnabled();
    userEvent.click(acceptBox);
    userEvent.click(submitButton);
    await waitFor(() => {
        expect(utils.getByRole('alert')).toBeVisible();
    });

    spy.mockRestore();
  });

});
