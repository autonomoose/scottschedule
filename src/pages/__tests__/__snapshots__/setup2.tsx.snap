// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SetupStep2Page renders snapshot correctly 1`] = `
<div
  data-testid="authenticator"
>
   
  <div
    style="margin: 1rem auto; min-height: 100vh; text-align: center;"
  >
    <div
      class="MuiBox-root css-0"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAppBar-root MuiAppBar-colorTransparent MuiAppBar-positionFixed mui-fixed css-1bzzxac-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1ti0wu3-MuiToolbar-root"
        >
          <button
            aria-label="Open menu"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeStart MuiIconButton-sizeMedium css-1t7w49f-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="MenuIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
            href="/home"
          >
            <div
              class="MuiBox-root css-k008qs"
            >
              <img
                alt="Werner Digital"
                src="../images/wernerdigital-hosted.png"
                width="26"
              />
              <p
                class="MuiTypography-root MuiTypography-body1 css-4peds9-MuiTypography-root"
              >
                Scottschedule
              </p>
            </div>
          </a>
          <div
            class="MuiBox-root css-yukqif"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
              href="/usermaint"
              style="margin-right: 1rem;"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                mockuname
              </p>
            </a>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1vbvugf-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Sign Out
            </button>
          </div>
          <button
            aria-label="Dark/light mode"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeStart MuiIconButton-sizeMedium css-1t7w49f-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="LightModeIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zM2 13h2c.55 0 1-.45 1-1s-.45-1-1-1H2c-.55 0-1 .45-1 1s.45 1 1 1zm18 0h2c.55 0 1-.45 1-1s-.45-1-1-1h-2c-.55 0-1 .45-1 1s.45 1 1 1zM11 2v2c0 .55.45 1 1 1s1-.45 1-1V2c0-.55-.45-1-1-1s-1 .45-1 1zm0 18v2c0 .55.45 1 1 1s1-.45 1-1v-2c0-.55-.45-1-1-1s-1 .45-1 1zM5.99 4.58c-.39-.39-1.03-.39-1.41 0-.39.39-.39 1.03 0 1.41l1.06 1.06c.39.39 1.03.39 1.41 0s.39-1.03 0-1.41L5.99 4.58zm12.37 12.37c-.39-.39-1.03-.39-1.41 0-.39.39-.39 1.03 0 1.41l1.06 1.06c.39.39 1.03.39 1.41 0 .39-.39.39-1.03 0-1.41l-1.06-1.06zm1.06-10.96c.39-.39.39-1.03 0-1.41-.39-.39-1.03-.39-1.41 0l-1.06 1.06c-.39.39-.39 1.03 0 1.41s1.03.39 1.41 0l1.06-1.06zM7.05 18.36c.39-.39.39-1.03 0-1.41-.39-.39-1.03-.39-1.41 0l-1.06 1.06c-.39.39-.39 1.03 0 1.41s1.03.39 1.41 0l1.06-1.06z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </header>
      <div
        class="MuiDrawer-root MuiDrawer-docked css-7ik032-MuiDrawer-docked"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiDrawer-paper MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft css-1hrb81m-MuiPaper-root-MuiDrawer-paper"
          style="visibility: hidden; webkit-transform: translateX(-0px); transform: translateX(-0px);"
        >
          <div
            class="MuiBox-root css-1ecnymn"
          >
            <div
              class="MuiBox-root css-xwg089"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
              >
                MainÂ Menu
              </h6>
              <button
                aria-label="Close menu"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1drgtl0-MuiButtonBase-root-MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="ArrowDropUpIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="m7 14 5-5 5 5z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-rr7dug-MuiDivider-root"
          />
          <ul
            class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
              href="/home"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-ltlrkn-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemIcon-root css-qklaqr-MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="HomeIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Home
                  </span>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
            </a>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
              href="/usermaint"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-ltlrkn-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemIcon-root css-qklaqr-MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Account
                  </span>
                </div>
              </div>
            </a>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
              href="/scheds"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-ltlrkn-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemIcon-root css-qklaqr-MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Schedules
                  </span>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
            </a>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
              href="/events"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-ltlrkn-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemIcon-root css-qklaqr-MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Events
                  </span>
                </div>
              </div>
            </a>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-14b5yu0-MuiTypography-root-MuiLink-root"
              href="/help"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-ltlrkn-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemIcon-root css-qklaqr-MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="StarIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Help
                  </span>
                </div>
              </div>
            </a>
            <div
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-ltlrkn-MuiButtonBase-root-MuiListItem-root"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root css-qklaqr-MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="StarIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-tlelie-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                >
                  Logout
                </span>
              </div>
            </div>
          </ul>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-h8m1q6"
    >
      <main>
        <div
          data-testid="mainPageDisplay"
          style="text-align: left;"
        >
          <div
            class="MuiBox-root css-1yp4ln"
          >
            <div
              class="MuiBox-root css-d0uhtl"
            >
              <img
                alt=""
                aria-labelledby="page-name"
                loading="eager"
                src="../images/wernerdigital-hosted.png"
                width="160"
              />
            </div>
            <div
              class="MuiBox-root css-1py79td"
            >
              <h1
                id="page-name"
              >
                WBench Signup
              </h1>
              <div
                class="MuiBox-root css-95qj54"
              >
                <span
                  style="color: rgb(119, 119, 119);"
                >
                  Step 1: Create account (Done!)
                </span>
              </div>
              <div
                class="MuiBox-root css-vx5vr9"
              >
                Step 2: Please read and accept our 
                <a
                  href="https://www.wernerdigital.com/terms"
                  style="color: rgb(0, 0, 238); text-decoration: underline;"
                >
                   Terms of Service 
                </a>
                <div
                  class="MuiBox-root css-d0uhtl"
                >
                  <form>
                    <br />
                     
                    <label
                      aria-label="acceptbox"
                      class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-jzcd7z-MuiFormControlLabel-root"
                    >
                      <span
                        class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary css-1e6ud36-MuiButtonBase-root-MuiCheckbox-root"
                      >
                        <input
                          class="PrivateSwitchBase-input css-1m9pwf3"
                          data-indeterminate="false"
                          name="acceptbox"
                          type="checkbox"
                        />
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                          data-testid="CheckBoxOutlineBlankIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </span>
                      <span
                        class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                      >
                        I accept!
                      </span>
                    </label>
                    <button
                      class="MuiButtonBase-root Mui-disabled MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall css-106f7v8-MuiButtonBase-root-MuiButton-root"
                      disabled=""
                      tabindex="-1"
                      type="submit"
                    >
                       Submit
                    </button>
                    <br />
                    <br />
                  </form>
                  (this will sign you out so that the changes can take effect)
                </div>
              </div>
              <div
                class="MuiBox-root css-95qj54"
              >
                Step 3: Sign back in to build/import schedules (or just run the clocks!)
              </div>
            </div>
          </div>
        </div>
      </main>
    </div>
    <footer
      style="padding-top: 40px;"
    >
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-rr7dug-MuiDivider-root"
      />
      <div
        class="MuiBox-root css-1bnxa12"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-yddyga-MuiTypography-root-MuiLink-root"
          href="/home"
        >
          Home
        </a>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-yddyga-MuiTypography-root-MuiLink-root"
          href="/usermaint"
        >
          Account
        </a>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-yddyga-MuiTypography-root-MuiLink-root"
          href="/scheds"
        >
          Schedules
        </a>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-yddyga-MuiTypography-root-MuiLink-root"
          href="/events"
        >
          Events
        </a>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-yddyga-MuiTypography-root-MuiLink-root"
          href="/help"
        >
          Help
        </a>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-rr7dug-MuiDivider-root"
      />
      <span
        class="MuiTypography-root MuiTypography-caption css-3merus-MuiTypography-root"
      >
        Â© 2021-
        2022
        , Werner Digital Technology Inc
      </span>
    </footer>
  </div>
</div>
`;
